src/components/content/style.css

```diff
@@ -0,0 +1,47 @@
+@value red, turquoise from "../variables/colors.css";
+
+.content {
+
+}
+
+.content a {
+  color: red;
+  white-space: nowrap;
+}
+
+.content a:hover {
+  color: turquoise;
+}
+
+.p {
+  composes: defaultFont from "../fonts/style.css";
+  margin: 1em 0;
+  line-height: 1.35;
+}
+
+.h1 {
```

Comment:
If we make our own components for headings (`H1`, `H2`, etc.) we can include this CSS there. Same for buttons too.

This will be useful, for example, when the content is data-driven and a particular heading needs to be centred. We can pass it as a prop and that `H1` component can apply the correct CSS class.